templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -806,6 +836,7 @@ Resources:
                   - "ATL_JDBC_DB_NAME=bitbucket"
                   - "ATL_JDBC_DRIVER=org.postgresql.Driver"
                   - "ATL_JDBC_USER=atlbitbucket"
+                  - !Sub ["ATL_BB_ADMIN_PASSWORD='${BitbucketAdminPassword}'",BitbucketAdminPassword: !If [IsBBAdminPassProvided, !Ref BitbucketAdminPassword, !Ref DBPassword ]]
```

Comment:
/nit missing comma and extra space before ending squared brackets:

`'${BitbucketAdminPassword}'", BitbucketAdminPassword`